tests/DetailTest/Persistence/Options/ModuleOptionsTest.php

```diff
@@ -43,4 +43,16 @@ public function testDoctrineDatetimeNoTzTypeCanBeSet()
 
         $this->assertTrue($this->options->getDoctrine()->registerDatetimeNoTzType());
     }
+
+    public function testDoctrineDatetimeImmutableNoTzTypeCanBeSet()
```

Comment:
All e tests in this class are doctrine options tests, I should refactor the Class and filename